eventing/samples/cronjob-source/README.md

```diff
@@ -0,0 +1,58 @@
+# Cron Job Source example
+
+Cron Job Source example shows how to configure Cron Job as event source for functions.
```

Comment:
I recommend using http://prettier.io/ to wrap this automatically with proseWrap=always (it makes it easier to make comments at the sub-paragraph level).